packages/slides-frontend/src/containers/redux-store/DeckReducer/reducer.ts

```diff
@@ -37,6 +37,43 @@ const DeckState = (state: Deck = initialDeck, action: Action): Deck =>
           ID: uuidv4(),
           itemsArray: [], // add a box that can be image or text
         }
+
+        // this code is to add two text Boxes by default in any new slide
+        // START
+        const TitleBox = {
+          type: ItemTypes.TEXT,
+        }
+        const DescriptionBox = {
+          type: ItemTypes.TEXT,
+        }
+        const itm1: Item = newItem(TitleBox);
+        const itm2: Item = newItem(DescriptionBox);
+
+        // itm1 - Title
+        itm1.Position = {
+          x: 0.30,
+          y: 0.20,
+        }
+        itm1.Size = {
+          width: 0.25,
+          height: 0.05,
+        }
+
+        // itm2 - Description
+        // this weird number comes from this calculation: title's med is: 30 + 25/2 = 42.5
+        // to reach this percentage and have the texts inline, the description's x + width/2 = 42.5
+        // width/2 is 20so x is 22.5 or 0.225
+        itm2.Position = {
+          x: 0.225,
+          y: 0.30,
+        }
+        itm2.Size = {
+          width: 0.40,
+          height: 0.40,
+        }
+        slide.itemsArray.push(itm1);
+        slide.itemsArray.push(itm2);
```

Comment:
you can just do it in one call to push - `slide.itemsArray.push(itm1, itm2)`

Reply:
:+1: